大一/数据结构/习题1/test1.cpp

```diff
@@ -0,0 +1,20 @@
+﻿#include <iostream>
+using namespace std;
+
+void hanoi(int n, char a, char c, char b) {
+    if (n == 1) {
+        cout << "移动盘 1 由 " << a << " -> " << c << endl;
+        return ;
+    }
+    hanoi(n - 1, a, b, c);
+    cout << "移动盘 " <<  n  << " 由 " << a << " -> " << c << endl;
+    hanoi(n - 1, b, c, a);
+}
+
+int main() {
```

Comment:
Usually we always define `void` in function with no args for better compatibility.